core/src/main/java/com/amplifyframework/predictions/models/CelebrityDetails.java

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.models;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.amplifyframework.util.Immutable;
+
+import java.net.URL;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Detailed metadata on a celebrity detection result.
+ */
+public final class CelebrityDetails {
+    private final Celebrity celebrity;
+    private final Pose pose;
+    private final List<FacialFeature> facialFeatures;
+    private final List<URL> urls;
+
+    private CelebrityDetails(final Builder builder) {
+        this.celebrity = builder.getCelebrity();
+        this.pose = builder.getPose();
+        this.facialFeatures = builder.getfacialFeatures();
```

Comment:
nit, --> should be getFactialFeatures with the first F as capital

Reply:
ah shoot. "Replace All" strikes again haha. will fix.